designs/developer/search-45/index.md

```diff
@@ -0,0 +1,9 @@
+---
+parent: Developer
+version: 4.5
+---
+# Search
+The Search page is the same page as displayed in the Administrator perspective.
+
+![Search page](img/image4.png)
```

Comment:
This page will need to be enhanced, I think you can pull this reference to the image here, so it doesn't need to be added.  What do you think @beaumorley ?

Reply:
@serenamarie125 This was a really old page I copied over. I was thinking we need to verify that that the img/image4.png image is still relevant (did not do that part) and then enhance this page with the Pin functionality. I think you are saying we can just delete the img/image4.png all together. That makes sense as it is probably out of date. I just copied the file but did not edit. We need to refine the process here (the way I did it for this first round may be more confusing!).

Reply:
understood!  If you're going to stage the site incrementally, you still need to make sure that the PR links and images will be valid, since subsequent PRs won't be up and merged immediately. 

So there are a couple of possibilities I can think of - you can either change to reference previous release, or just say coming soon until we get the up to date docs!

Reply:
OK. I understand your point. I changed this one to coming soon. I'll be curious to hear from the team if this works for them as a process.